src/txn/txn_rollback_to_stable.c

```diff
@@ -926,6 +921,14 @@ __rollback_to_stable_btree_hs_cleanup(WT_SESSION_IMPL *session, uint32_t btree_i
 
         /* Set this comparison as exact match of the search for later use. */
         cbt->compare = 0;
+        WT_ERR(hs_cursor->get_value(hs_cursor, &hs_stop_ts, &durable_ts, &type_full, hs_value));
```

Comment:
This seems expensive for cases where verbose isn't enabled - can you only retrieve the value if `WT_VERB_RTS` is true?

Reply:
I reduced the verbose scope as not many details are not required in that function.